inc/log.class.php

```diff
@@ -483,6 +483,35 @@ static function getHistoryData(CommonDBTM $item, $start=0, $limit=0, $sqlfilter=
                   }
                   $tmp['change'] = sprintf(__('%1$s: %2$s'), __('Add a link with an item'),
                                            $data["new_value"]);
+              
+                  if ($data['itemtype'] == 'Ticket') {
+                     if ($data['itemtype_link'] == 'Group') {
+                        $itemtick = new Group_Ticket();
+                     } else if  ($data['itemtype_link'] == 'User') {
+                        $itemtick = new Ticket_User();
+                     } else if  ($data['itemtype_link'] == 'Supplier') {
+                        $itemtick = new Supplier_Ticket();
+                     }
+                     $table   = $itemtick->getTable();
+                     $key     = getForeignKeyFieldForItemType($data['itemtype']);
+                     $itemkey = getForeignKeyFieldForItemType($data['itemtype_link']);
+                     $iditem  = trim(substr($data['new_value'], strrpos($data['new_value'], '(')+1,
+                                     strrpos($data['new_value'], ')')),')');
+
+                     foreach($DB->request($table, array($key => $data['items_id'],
+                                                        $itemkey => $iditem)) as $datalink) {
+                        if ($datalink['type'] == 1) {
```

Comment:
Please use CommonITILActor::REQUESTER and other constants instead of direct values ;)